packages/workspace/src/common/search-service.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2017 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+export const searchServicePath = '/services/search';
+
+/**
+ * The JSON-RPC search service interface.
+ */
+export const SearchService = Symbol('SearchService');
+export interface SearchService {
```

Comment:
maybe `WorkspaceSearchService`, otherwise it sounds very generic

Reply:
How about `FileSearchService`? The service can search anywhere (first param) not only in workspace.

Reply:
+1